.travis.yml

```diff
@@ -14,16 +14,13 @@ cache:
   - ccache
 
 env:
-  - THREADING="serial"
-  - THREADING="openmp"
-  - THREADING="pthread"
+  - 
```

Comment:
Why is this line empty

Reply:
This basically means no environment variables.